content/en/blog/_posts/2022-12-12-host-process-containers-stable/index.md

```diff
@@ -0,0 +1,113 @@
+the host's network namespace and access/configure storage mounts visible to the host.
+On the other hand, Windows Server containers are highly isolated and exist in a separate
+silo'd namespace. Direct access to the host from a Windows Server container is explicitly disallowed
+by default. Mounting a host volume into the Windows Server container is highly discouraged to minimize
+potential attack surfaces. HostProcess containers enable you to configure the host such that communication
+with your Windows container workloads remains within the bounds of a secure pipeline.
+
+Until now, scenarios common to Linux privileged containers, such as kube-proxy (via kubeadm),
+storage, and networking, all required proxy solutions to enable functionality on Windows.
+With HostProcess containers users have direct access to the node without the need for
+any workarounds. Gone are the days where cluster operators would need to manually log onto
+Windows nodes to perform administrative duties - instead replaced with a continuous
+development pipeline.
+
+## How does it work?
+
+Windows HostProcess containers are implemented with Windows _Job Objects_, a break from the
+previous container model using server silos. Job objects are components of the Windows OS which offer the ability to
+manage a group of processes as a group (a.k.a. _jobs_) and assign resource constraints to the
+group as a whole. Job objects are specific to the Windows OS and are not associated with the Kubernetes [Job API](https://kubernetes.io/docs/concepts/workloads/controllers/job/). They have no process or file system isolation,
+enabling the privileged payload to view and edit the host file system with the
+correct permissions, among other host resources. The init process, and any processes
+it launches or that are explicitly launched by the user, are all assigned to the
+job object of that container. When the init process exits or is signaled to exit,
+all the processes in the job will be signaled to exit, the job handle will be
+closed and the storage will be unmounted.
+
+HostProcess and Linux privileged containers enable similar scenarios but differ
+greatly in their implementation (hence the naming difference). HostProcess containers
+have their own pod security policies. Those used to configure Linux privileged
+containers **do not** apply. Enabling privileged access to a Windows host is a
+fundamentally different process than with Linux so the configuration and
+capabilities of each differ significantly. Below is a diagram detailing the
+overall architecture of Windows HostProcess containers:
+
+{{< figure src="hpc-architecture.png" alt="HostProcess Architecture" >}}
+
+Two major features were added prior to moving to stable: the ability to run as local user accounts, and
+a simplified method of accessing volume mounts. For more info visit our [updated documentation page](/docs/tasks/configure-pod-container/create-hostprocess-pod)
+
+## Use Cases
```

Comment:
(nit)
```suggestion
## Use cases
```

Reply:
Reading through I thought I was going to learn about use cases such as "CNI's, kube-proxy, configuration management, etc"  but below is a list of projects using HPC's.  My consider renaming to something like "Hostprocess contianers in action" and making this section a bulleted list to make the projects stand out.

Reply:
Either way works